settings.gradle

```diff
@@ -1,5 +1,6 @@
 pluginManagement {
     repositories {
+        mavenLocal()
```

Comment:
sonatype?

Reply:
`mavenCentral()` ?

Reply:
I was referring to sonatype, as there was the snapshot. With the release, no change is required.